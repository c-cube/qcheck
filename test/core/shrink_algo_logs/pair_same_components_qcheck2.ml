open QCheck2

let pair_same =
  Test.make ~name:"pairs have same components"
    Gen.(pair int int)
    (Log.shrinks Print.(pair int int) @@
      fun (i,j) -> i=j)
;;
try Test.check_exn pair_same with Test.Test_fail _ -> ()
