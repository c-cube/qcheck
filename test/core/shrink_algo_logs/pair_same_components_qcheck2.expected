fails (-4571327332697646483, -3285013039971199785)
fails (2732179349098564728, -3285013039971199785)
fails (0, -3285013039971199785)
fails (0, 1281206205950578423)
holds (0, 0)
fails (0, 640603102975289211)
holds (0, 0)
fails (0, 320301551487644605)
holds (0, 0)
fails (0, 160150775743822302)
holds (0, 0)
fails (0, 80075387871911151)
holds (0, 0)
fails (0, 40037693935955575)
holds (0, 0)
fails (0, 20018846967977787)
holds (0, 0)
fails (0, 10009423483988893)
holds (0, 0)
fails (0, 5004711741994446)
holds (0, 0)
fails (0, 2502355870997223)
holds (0, 0)
fails (0, 1251177935498611)
holds (0, 0)
fails (0, 625588967749305)
holds (0, 0)
fails (0, 312794483874652)
holds (0, 0)
fails (0, 156397241937326)
holds (0, 0)
fails (0, 78198620968663)
holds (0, 0)
fails (0, 39099310484331)
holds (0, 0)
fails (0, 19549655242165)
holds (0, 0)
fails (0, 9774827621082)
holds (0, 0)
fails (0, 4887413810541)
holds (0, 0)
fails (0, 2443706905270)
holds (0, 0)
fails (0, 1221853452635)
holds (0, 0)
fails (0, 610926726317)
holds (0, 0)
fails (0, 305463363158)
holds (0, 0)
fails (0, 152731681579)
holds (0, 0)
fails (0, 76365840789)
holds (0, 0)
fails (0, 38182920394)
holds (0, 0)
fails (0, 19091460197)
holds (0, 0)
fails (0, 9545730098)
holds (0, 0)
fails (0, 4772865049)
holds (0, 0)
fails (0, 2386432524)
holds (0, 0)
fails (0, 1193216262)
holds (0, 0)
fails (0, 596608131)
holds (0, 0)
fails (0, 298304065)
holds (0, 0)
fails (0, 149152032)
holds (0, 0)
fails (0, 74576016)
holds (0, 0)
fails (0, 37288008)
holds (0, 0)
fails (0, 18644004)
holds (0, 0)
fails (0, 9322002)
holds (0, 0)
fails (0, 4661001)
holds (0, 0)
fails (0, 2330500)
holds (0, 0)
fails (0, 1165250)
holds (0, 0)
fails (0, 582625)
holds (0, 0)
fails (0, 291312)
holds (0, 0)
fails (0, 145656)
holds (0, 0)
fails (0, 72828)
holds (0, 0)
fails (0, 36414)
holds (0, 0)
fails (0, 18207)
holds (0, 0)
fails (0, 9103)
holds (0, 0)
fails (0, 4551)
holds (0, 0)
fails (0, 2275)
holds (0, 0)
fails (0, 1137)
holds (0, 0)
fails (0, 568)
holds (0, 0)
fails (0, 284)
holds (0, 0)
fails (0, 142)
holds (0, 0)
fails (0, 71)
holds (0, 0)
fails (0, 35)
holds (0, 0)
fails (0, 17)
holds (0, 0)
fails (0, 8)
holds (0, 0)
fails (0, 4)
holds (0, 0)
fails (0, 2)
holds (0, 0)
fails (0, 1)
holds (0, 0)
